ocs_ci/utility/kms.py

```diff
@@ -2059,6 +2067,44 @@ def verify_osd_keys_present_on_azure_kv(self):
         logger.info("All OSD keys are present in the Azure KV ")
         return True
 
+    def remove_kmsid(self):
+        """
+        Removing azure kmsid from the configmap `csi-kms-connection-details`.
+        """
+        if not self.is_azure_kv_connection_exists():
+            logger.info(
+                f"There is no KMS connection {self.azure_kms_connection_name} available in the configmap"
+            )
+            return False
+
+        csi_kms_configmap = ocp.OCP(
+            kind=constants.CONFIGMAP,
+            resource_name=constants.VAULT_KMS_CSI_CONNECTION_DETAILS,
+            namespace=self.namespace,
+        )
+
+        if len(get_encryption_kmsid()) == 1:
```

Comment:
can there be situations there are  more than 1 KMS? How about covering the `>= 1` case?

Reply:
If the cluster is already configured with the Vault KMS then it will have multiple KMS IDs.